tests/unittest/bindings/test_transfer_agent_bindings.py

```
@@ -465,6 +465,90 @@ def test_nixl_write_transfer_multiple_chunks(self):
+        agent_b.register_memory(dst_descs)
+
+        agent_a.load_remote_agent("agent_b", agent_b.get_local_agent_desc())
+        agent_b.load_remote_agent("agent_a", agent_a.get_local_agent_desc())
+
+        # Invalidate the remote agent before submitting
+        agent_a.invalidate_remote_agent("agent_b")
+
+        request = tab.TransferRequest(tab.TransferOp.WRITE, src_descs, dst_descs, "agent_b")
+
+        # Submitting to an invalidated remote should raise or return a failed status
+        with pytest.raises(Exception):
```

Ruff (0.15.1)

[warning] Do not assert blind exception: `Exception`

(B017)